src/Uno.UI/ts/Windows/ApplicationModel/Contacts/ContactPicker.ts

```diff
@@ -0,0 +1,50 @@
+﻿enum ContactProperty {
+	Address = "address",
+	Email = "email",
+	Icon = "icon",
+	Name = "name",
+	Tel = "tel"
+};
+
+
+declare class ContactInfo {
+	address: PaymentAddress[];
```

Comment:
Codacy found an issue: [The class property 'address' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6378321)